function/services/application-mapper/index.test.js

```diff
@@ -307,4 +307,539 @@ describe('Application mapper', () => {
+                ]
+            };
+
+            oracleObject = await mapApplicationDataToOracleObject(
+                applicationSummaryJson,
+                applicationFormDefault,
+                addressDetailsDefault
+            );
+            applicationFormJson = Object.values(oracleObject)[0][0].APPLICATION_FORM;
+
+            expect(applicationFormJson.previous_application_submitted).toBe('Y');
+            expect(applicationFormJson.prev_app_for_ci_comp).toBe('Y');
+        });
+
+        it('Should set columns to "Y" if answered Yes to someone-else-applied-before and applied-to-us-before answered No', async () => {
+            applicationSummaryJson = {
+                meta: {
+                    caseReference: '23\\327507',
+                    submittedDate: '2023-05-19T13:06:12.693Z',
+                    splitFuneral: false
+                },
+                themes: [
+                    {
+                        type: 'theme',
+                        id: 'about-application',
+                        title: 'About your application',
+                        values: [
+                            {
+                                id: 'q-applicant-applied-before-for-this-crime',
+                                type: 'simple',
+                                value: false,
+                                theme: 'about-application'
+                            },
+                            {
+                                id: 'q-applicant-someone-else-applied-before-for-this-crime',
+                                type: 'simple',
+                                value: 'yes',
+                                theme: 'about-application'
+                            }
+                        ]
+                    },
+                    {
+                        type: 'theme',
+                        id: 'other-compensation',
+                        title: 'Other compensation',
+                        values: [
+                            {
+                                id: 'q-applicant-have-you-applied-to-us-before',
+                                type: 'simple',
+                                value: false,
+                                theme: 'other-compensation'
+                            }
+                        ]
+                    }
+                ]
+            };
+
+            oracleObject = await mapApplicationDataToOracleObject(
+                applicationSummaryJson,
+                applicationFormDefault,
+                addressDetailsDefault
+            );
+            applicationFormJson = Object.values(oracleObject)[0][0].APPLICATION_FORM;
+
+            expect(applicationFormJson.previous_application_submitted).toBe('Y');
+            expect(applicationFormJson.prev_app_for_ci_comp).toBe('Y');
+        });
+
+        it('Should set columns to "N" if answered I dont know to someone-else-applied-before and applied-to-us-before answered Yes', async () => {
```

Comment:
test functionality is correct but named incorrectly, it should set columns to "Y" not "N"